addons/armory/XEH_postInit.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+// Talking Armory Boxes
+[QGVAR(Locker), "containerOpened", {
+    [QGVAR(talkingBoxEvent), [_this select 0]] call CBA_fnc_serverEvent;
+}, true, [], true] call CBA_fnc_addClassEventHandler;
+
+[QGVAR(talkingBoxEvent), FUNC(talkingBox)] call CBA_fnc_addEventHandler;
+
+GVAR(boxLines) = [
+    "tacr_box_100_quid",
```

Comment:
This should be done like `radios` component does it. Cache all sounds based on their config-entry in [`preInit`](https://github.com/Theseus-Aegis/Mods/blob/29a1586aab37d5dfde462a7c22edb979b7668947/addons/radios/XEH_preInit.sqf#L12).